_includes/search-loader.html

```diff
@@ -14,7 +14,7 @@
 </div>
 {% endcapture %}
 
-{% capture not_found %}<p class="mt-5">Oops! No result founds.</p>{% endcapture %}
+{% capture not_found %}<p class="mt-5">{{ site.data.locales[lang].search.loader}}</p>{% endcapture %}
```

Comment:
Please rename the variable `*.search.loader` as above, and don't forget to put a space before the symbol `}}`